fable-test-app/src/pages/ViewHolidaysNation.tsx

```diff
@@ -0,0 +1,131 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
+// Components (internal)
+import { DateModified, Heading, Select } from '../components';
+import { formatDate } from '../utils/utils';
+
+const ViewHolidaysNation = () => {
+  // State variables
+  const [holidays, setHolidays] = useState([]);
+  const [year, setYear] = useState<string>('2024');
+  const [yearsList] = useState<string[]>(['2022', '2023', '2024', '2025', '2026']);
```

Comment:
Same here, could we move these values into a config file somewhere or defined constants? I think Paul's API currently has support for up to 2030. I wonder if we can show current year +/- 2 to 3 years?